### ISSUE TITLE
Randomly Fails to Deserialize TOML File

### ISSUE THREAD

**Issue by TheCedarPrince** [greshake/i3status-rust#1806]
Hi folks,

I have been using i3status-rust for over a year or so without any issues whatsoever -- it's a fantastic piece of software! This morning, I booted up my laptop and for some reason, I am now getting this error with i3status-rust even though I did no changes to my config:

```sh 
{"version": 1, "click_events": true}
[
[{"full_text":"Failed to deserialize TOML file ~/.config/i3status-rust/config.toml at line 4: unknown field`buttons`","instance":"0:","separator":false,"separator_block_width":0,"markup":"pango"}],


Failed to deserialize TOML file ~/.config/i3status-rust/config.toml at line 4: unknown field `buttons`


[src/main.rs:115] error = Error {
    kind: Other,
    message: Some(
        "Failed to deserialize TOML file ~/.config/i3status-rust/config.toml at line 4: unknown field `buttons`",
    ),
    cause: None,
    block: None,
}
```

And here is my config:

```toml
theme = "space-villain"
icons = "material-nf"

[[block]]
block = "music"
buttons = ["play", "next"]
interface_name_exclude = ['.*kdeconnect.*', 'mpd']

[[block]]
block = "pomodoro"
length = 25
break_length = 5
message = "Take a break!"
break_message = "Back to work!"
notifier = "notifysend"
[block.theme_overrides]
idle_bg = "#420D09"

[[block]]
block = "time"
interval = 5
format = "%R"
[block.theme_overrides]
idle_bg = "#21201E"

[[block]]
block = "sound"
[block.theme_overrides]
idle_bg = "#301934"

[[block]]
block = "battery"
interval = 10
format = "{percentage}"
[block.theme_overrides]
good_bg = "#013220"

# [[block]]
# block = "networkmanager"
# interface_name_exclude = ['br\-[0-9a-f]{12}', 'docker\d+']
# interface_name_include = []
# ap_format = "{strength}"
# device_format = "{icon} {ap}"
```

Additionally, here is my OS information:

```
NAME="Red Hat Enterprise Linux"
VERSION="8.7 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.7"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.7 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8::baseos"
HOME_URL="https://www.redhat.com/"
DOCUMENTATION_URL="https://access.redhat.com/documentation/red_hat_enterprise_linux/8/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"
REDHAT_BUGZILLA_PRODUCT_VERSION=8.7
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.7"
```

And my i3status-rust version is: `i3status-rs 0.30.2`

It seems very similar to the issue that was just opened and closed #1805 by @NA3. 

Any thoughts as to what is going on? Again, I have used this config for over a year or so and it has caused no issues until today. Thanks!

**Comment by MaxVerevkin** [comment]
v0.30 has quite a lot of breaking changes (see [NEWS.md](https://github.com/greshake/i3status-rust/blob/master/NEWS.md#i3status-rust-0300)), so the config needs to be adapted.

**Comment by TheCedarPrince** [comment]
@MaxVerevkin -- ah! That was it! I have nearly upgraded and fixed everything on my config. The only thing I still have a problem with is pomodoro. I tried looking at the documentation and the updates, but I no longer have an icon to click and a timer shown. I tried playing with the `format` string, but to no avail in fixing this. Here is my revised config:

```toml
[theme]
theme = "space-villain"
[icons]
icons = "material-nf"

[[block]]
block = "music"
format = " $icon $play $next {$combo.str(max_w:40,rot_interval:0.5)} |"
interface_name_exclude = ['.*kdeconnect.*', 'mpd']

[[block]]
block = "pomodoro"
notify_cmd = "$icon notify-send '{msg}'"
message = "Take a break!"
break_message = "Back to work!"
blocking_cmd = false
format = " $icon $pomodoro"
[block.theme_overrides]
idle_bg = "#420D09"

[[block]]
block = "time"
interval = 5
[block.format]
short = " $icon $timestamp.datetime(f:%R) "
[block.theme_overrides]
idle_bg = "#21201E"

[[block]]
block = "sound"
[block.theme_overrides]
idle_bg = "#301934"

[[block]]
block = "battery"
interval = 10
format = " $icon $percentage"
[block.theme_overrides]
good_bg = "#013220"
```

Basically, I am after the functionality I had where there was a play/pause button to start the timer, show the timer, and always show the timer. Would you be able to help me with this? Thanks and thanks for the rapid response! :smile:

**Comment by MaxVerevkin** [comment]
@TheCedarPrince can you try the most basic 

```toml
[[block]]
block = "pomodoro"
```

?

**Comment by TheCedarPrince** [comment]
@MaxVerevkin , here is what I see:

![image](https://user-images.githubusercontent.com/29561456/222459029-cd175014-d0c6-4b1c-907c-9b9369331a12.png)

No timer or clickable icons. Just the tomato icon

**Comment by MaxVerevkin** [comment]
Tomato is clickable :)

**Comment by TheCedarPrince** [comment]
Hm. I just tried clicking it and it doesn't seem to do anything. I thought an icon should change or something?

**Comment by MaxVerevkin** [comment]
> i3status-rs 0.30.2

Ah, that's the problem. This was fixed in v0.30.3.

**Comment by TheCedarPrince** [comment]
Ahhhhhh got it. I'll have to wait until COPR rebuilds and tags the latest release.

Either way, thanks for the help! This is a great tool. Hope you have a wonderful day @MaxVerevkin!!!
